README.md

```diff
@@ -76,4 +76,26 @@ make test
 ```
 /!\ Warning: Running tests requires `go`, `php8.1` and `composer` installed globally.
 
-/!\ Warning: You will also need the following PHP extensions installed: `php8.1-xml`, `php8.1-mbstring` and `php8.1-curl`. 
+/!\ Warning: You will also need the following PHP extensions installed: `php8.1-xml`, `php8.1-mbstring`, `php8.1-zip` and `php8.1-curl`. 
+
+
+## End to End testing
+
+You will need to have `Firefox` installed and add the following configuration lines to the `webapp/.env` file:
+
+```sh
+# The webapp URL
+WEBAPP_URL="http://localhost:9000"
+```
+
+The you need to run the full app stack in development mod using:
+
+```sh
+make run
+```
+
+You can the  run end to end tests using:
+
+```sh
+make test-e2e
+```
```

Comment:
`make run` works great, and I love the new UI! :heart: 

However, I'm running into some issues when trying to launch the E2E tests:

```
Testing /home/slax57/workspaces/labyrinth/webapp/tests/E2E
EE                                                                  2 / 2 (100%)

Time: 00:05.291, Memory: 18.00 MB

There were 2 errors:

1) App\Tests\E2E\BoardTest::testRotateRemainingTileClockwise
Facebook\WebDriver\Exception\ElementClickInterceptedException: Element <button id="submit-rotate-remaining-clockwise" type="submit"> is not clickable at point (1504,715) because another element <div id="sfToolbarMainContent-740920" class="sf-toolbarreset notranslate clear-fix"> obscures it

/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Exception/WebDriverException.php:96
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/HttpCommandExecutor.php:359
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:605
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteExecuteMethod.php:23
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebElement.php:79
/home/slax57/workspaces/labyrinth/webapp/vendor/symfony/panther/src/Client.php:226
/home/slax57/workspaces/labyrinth/webapp/vendor/symfony/browser-kit/AbstractBrowser.php:313
/home/slax57/workspaces/labyrinth/webapp/tests/E2E/BoardTest.php:25

2) App\Tests\E2E\BoardTest::testRotateRemainingTileAntilockwise
Facebook\WebDriver\Exception\ElementClickInterceptedException: Element <button id="submit-rotate-remaining-anticlockwise" type="submit"> is not clickable at point (1326,715) because another element <div id="sfToolbarMainContent-74da51" class="sf-toolbarreset notranslate clear-fix"> obscures it

/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Exception/WebDriverException.php:96
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/HttpCommandExecutor.php:359
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:605
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteExecuteMethod.php:23
/home/slax57/workspaces/labyrinth/webapp/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebElement.php:79
/home/slax57/workspaces/labyrinth/webapp/vendor/symfony/panther/src/Client.php:226
/home/slax57/workspaces/labyrinth/webapp/vendor/symfony/browser-kit/AbstractBrowser.php:313
/home/slax57/workspaces/labyrinth/webapp/tests/E2E/BoardTest.php:50

ERRORS!
Tests: 2, Assertions: 2, Errors: 2.

Remaining indirect deprecation notices (4)
make[1]: *** [Makefile:17 : test-e2e] Erreur 2
```

Also, it would be nice to add another `Makefile` target, for instance `test-e2e-dev` with the necessary options enabled to make the debugging of E2E tests easier. For example following these guidelines: https://github.com/symfony/panther#interactive-mode